pkg/registries/doc.go

```diff
@@ -0,0 +1,5 @@
+// Package registries contains shared infrastructure to build/update /etc/containers/registries.conf
+// based on cluster-wide configuration.
+//
+// NOTE: This has users outside of the machine-config-operator repository!
```

Comment:
Who are they?  I can't turn them up in [GitHub's search][1] or [GoDoc][2], although neither of those are exhaustive.  But the idea that we'll have multiple repositories calling `EditRegistriesConfig` freaks me out a little ;).

[1]: https://github.com/search?q=github.com%2Fopenshift%2Fruntime-utils%2Fpkg%2Fregistries&type=Code
[2]: https://godoc.org/github.com/openshift/runtime-utils/pkg/registries?importers

Reply:
- This was copy&pasted from MCO, and the wording above, warning people not to change the API unaware, is no longer really pertinent — it’s neither necessary to warn so strongly, nor it’s really clear why this should refer to the MCO repository at all.
- `EditRegistriesConfig` is a memory-only operation; it does not change any on-disk configuration itself, it’s up to the caller to read the created data, or to write it on disk somewhere and make it used. E.g. https://github.com/openshift/openshift-controller-manager/pull/19 creates a config map entry that is then mounted into a container. It’s not an automatic invitation for all of the callers to race each other overwriting node-level configuration files (this repo neither exactly prevents them from doing so, nor does it make it any easier).